gallery/src/components/NxCollapsibleMultiSelect/NxCollapsibleMultiSelectCustomTooltipExample.tsx

```diff
@@ -16,6 +16,7 @@ import {
 } from '@sonatype/react-shared-components';
 
 interface CustomOption extends NxCollapsibleMultiSelectOption {
+  name: string;
```

Comment:
Let's add a comment explaining why we're including `name` here: to narrow its type to string

Reply:
> Let's add a comment explaining why we're including name here: to narrow its type to string

And let's add that same comment to the other examples that do the same thing.